.github/workflows/test.yml

```diff
@@ -5,20 +5,11 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest, macos-latest ]
+      fail-fast: false
     runs-on: ${{ matrix.os }}
     steps:
     - uses: actions/checkout@v2
     - uses: cachix/install-nix-action@v13
-      with:
-        # We need a nix version with NixOS/nix#3564 to be able to use
-        # `--max-jobs`, so using NixOS/nix@323e545. See 8365e73 for the problem
-        # with `--max-jobs`. Install URL is obtained from CI logs for the nix
-        # commit: https://github.com/NixOS/nix/runs/2895674848 expand "Run
-        # cachenix/install-nix-action@v13" job, then the "Run
-        # cachenix/install-nix-action@v13" step, and you'll see a line starting
-        # with `install_url`.
-        install_url: https://nixos-nix-install-tests.cachix.org/serve/i6laym9jw3wg9mw6ncyrk6gjx4l34vvx/install
-        install_options: '--tarball-url-prefix https://nixos-nix-install-tests.cachix.org/serve'
```

Comment:
Why is this removed?

Reply:
I am still experimenting. `nix` master segfauls on darwin, the released version doesn’t. But then we are of course hitting
```
error: all build users are currently in use; consider creating additional users and adding them to the 'nixbld' group
```
again. I’ll see if there is a way to create more users.

Reply:
Ping @nomeata